src/pages/LoginForm.js

```diff
@@ -27,9 +26,7 @@ export default function LoginForm() {
   return (
     <>
       <Globalstyles />
-
       <LoginFormWrapper>
-
         <LoginWrapper>
           <TextFieldContainer
             input
```

Comment:
You should use a placeholder and close the "TextFieldContainer" Tag.
I think "input" does not do anything here.